packages/v4/src/content/developer-resources/accessibility-testing.md

```diff
@@ -0,0 +1,106 @@
+
+- **Accessible names for all elements** - When an element does not have visual text or when further explanation is necessary, a screen reader will not know what an item is or does. If you have an icon `<button>`, for example, the screen reader can only tell that it’s a button if it doesn’t have a label, not what the button does. Add a few more non-labeled buttons, and now all the screen reader sees are vague buttons with no understanding of what they do. See above section on labeling for more information.
+
+- **Links** - similar to the above point, links need clarity too and should always have a label. Without it, users have no idea where the links point to - would you click on a link if you had no idea where it went? If you have the same label for multiple links though, it needs to be the same URL. Links, buttons, and form controls should make sense out of context. If a user wants to look at all of the links available, they should be able to differentiate the available links. You shouldn’t have ten “Click here” links.
+
+- **Landmarks** - As mentioned earlier, landmarks help identify regions to communicate the structure of a page. There are a number of types of landmarks such as banners, navigation, main, form, etc. When there are more than one of the same role (for example, two navigation regions on the same page), these regions should be differentiated by an `aria-label`.
+
+- **Dynamic content** - One of the biggest accessibility concerns with dynamic content is the need to notify users that the content has changed. Sighted users benefit from highlighting or drawing visual attention to the changes. Non-sighted users need to be notified some other way, such as by loading a new page, sending the focus to the new content, or using ARIA live announcements.
+
+- **Unrelated announcements** from the OS outside of the web app - Other notifications can interrupt a user while interacting with a page. For example, "you received a new chat msg from xyz". This possibility should be considered when designing and developing for screen readers as it will impact their experience. This is why it's good to keep things like validation messages and alerts terse, as it makes it less likely that an announcement will be interrupted. 
+
+
+<br />
+
+### Checklist for testing
+These are some of the main areas we check for on the PatternFly team. We recommend that you check for these same areas in your applications, too. 
+
+**General questions**
+
+<input type="checkbox" unchecked /> VO Rotor Navigation can discover all information <br/>
+<input type="checkbox" unchecked /> VO Shortcut Navigation can discover all information <br/>
+<input type="checkbox" unchecked /> VO Cursor Navigation can discover all information. (dependent on component and use case- easy way to navigate to component) <br/>
```

Comment:
```suggestion
<input type="checkbox" unchecked /> VO Cursor Navigation can discover all information (dependent on component and use case—easy way to navigate to component). <br/>
```  
I'm not quite sure what "easy way to navigate to component" means. Do you mean, "Users should have an easy way of navigating components" ?